README.md

```diff
@@ -1,39 +1,96 @@
+transports on the default port (`4062`):
 
-- [Slice Greeter](./src//slice/Greeter.slice) - `icerpc://hello.icerpc.dev/VisitorCenter.Greeter`
-  - [IceRPC C# Secure Slice example client](https://github.com/icerpc/icerpc-csharp/tree/0.3.x/examples/slice/Secure/Client)
-  - [IceRPC C# QUIC Slice example client](https://github.com/icerpc/icerpc-csharp/tree/0.3.x/examples/slice/Quic/Client)
-- [Protobuf Greeter](./src/proto/greeter.proto) - `icerpc://hello.icerpc.dev/visitor_center.Greeter`
-  - [IceRPC C# Secure Protobuf  example client](https://github.com/icerpc/icerpc-csharp/tree/0.3.x/examples/protobuf/Secure/Client)
-  - [IceRPC C# QUIC Protobuf example client](https://github.com/icerpc/icerpc-csharp/tree/0.3.x/examples/protobuf/Quic/Client)
+| Service                                       | Path                                               | Description                           | Example Clients                                                                                                                                                                                 |
+| --------------------------------------------- | -------------------------------------------------- | ------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Slice Greeter](./src//slice/Greeter.slice)   | `icerpc://hello.icerpc.dev/VisitorCenter.Greeter`  | A simple service that greets visitors | [C# Secure](https://github.com/icerpc/icerpc-csharp/tree/0.3.x/examples/slice/Secure/Client) <br>[C# QUIC](https://github.com/icerpc/icerpc-csharp/tree/0.3.x/examples/slice/Quic/Client)       |
+| [Protobuf Greeter](./src/proto/greeter.proto) | `icerpc://hello.icerpc.dev/visitor_center.Greeter` | A simple service that greets visitors | [C# Secure](https://github.com/icerpc/icerpc-csharp/tree/0.3.x/examples/protobuf/Secure/Client) <br>[C# QUIC](https://github.com/icerpc/icerpc-csharp/tree/0.3.x/examples/protobuf/Quic/Client) |
 
-## Running the server locally with Docker Compose
+## Running the server using Docker
 
-To run the server locally, you can use Docker Compose. The following command will start the server using the
-configuration in the [`docker-compose.yml`](./docker-compose.yml) file.
+The `hello` server is available as a Docker image on [Docker Hub](https://hub.docker.com/r/icerpc/hello). The server
+requires TLS certificates to run, so you will need to provide the server certificate and private key. Certificates can
+be obtained using a service like [Let's Encrypt](https://letsencrypt.org/). This repository also contains a set of
+self-signed [certificates](./certs) that can be used for testing purposes.
+
+### Docker CLI
 
 ```bash
-docker compose up
+docker run --name hello -p 4062:4062/tcp -p 4062:4062/udp -v /path/to/certificates/:/certs icerpc/hello
```

Comment:
I assume:
`-v /path/to/certificates/:/certs`

means mount the /path/to/certificates/ directory as /certs?

Is it common to use different names (certifications vs certs)?

> so you will need to provide the server certificate and private key

It's not clear to me which names I need to pick for these files.

Reply:
> means mount the /path/to/certificates/ directory as /certs?
Yes!

> Is it common to use different names (certifications vs certs)?

There's no preference either way. For instance the popular `linuxserver/nginx` container uses `-v /some/content:/usr/share/nginx/html:ro` as their example.

> It's not clear to me which names I need to pick for these files.

I'll elaborate